### DISCUSSION TITLE
I was dumb and now I'm paying for it

### DISCUSSION THREAD

**Discussion by Compodulator** [mastodon/mastodon#25664] [Q&A] [answered]
A while back I made two mastodon accounts.
Where?
**GOOD QUESTION!**

One was on social, but the other... I don't know...
So, I managed to access the social one... **AND PROCEEDED TO DELETE IT!**
Why? **GOOD QUESTION!**

So now I have to wait 30 days for it to get deleted.
**FINE!**
Will I be able to make a new account under the same name and email, though?

**Comment by YJSoft** [comment, marked as answer]
It's impossible to reuse handle(a.k.a. id) if deleted. however, emails should be reuseable.

**Reply by Compodulator** [reply]
Great. Lovely. Awesome.
Seeing as that handle was my "brand name", I might as well get a rope.

**Comment by Gankaku** [comment]
> Great. Lovely. Awesome. Seeing as that handle was my "brand name", I might as well get a rope.

@Compodulator You'll need to get creative. Like put The in front, or "online" after, or something like that. I can help you if you want! Or...tell ChatGPT your issue and what kind of names you'd like using your original ID. It will come up with something creative for you. good luck! If you keep your original, but have something after or before it, people will still know you.

**Reply by Compodulator** [reply]
Yeah, I ended up using TheCompodulator on social.
It still feels fake.
